bin/meta-git-remote

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const program = require('commander');
+const debug = require('debug')('meta-git-remote');
+const loop = require('loop-things');
+const path = require('path');
+const util = require('util');
+
+const usages = `\n  usages:\n
+    meta git remote
+    meta git remote -v
+    meta git remote show origin
+    meta git remote rename oldname newname
+    meta git remote rm name`;
+
+if (process.argv[2] === '--help')
+  return console.log(usages);
+
+let first = process.argv[2] === 'blank' ?
+                      process.argv[3] || '':
+                      process.argv[2] || '';
+
+const subCommandParsers = {
+  '-v': function (argv) { return '' },
+  'show': function (argv) {
+    let remote = process.argv[3] === 'blank' ?
+                      process.argv[4] || '':
+                      process.argv[3] || '';
+    return `show ${remote}`;
+  },
+  'rename': function (argv) {
+    
+  },
+  'rm': function (argv) {}
+}
+
+const subCommandParser = subCommandParsers[first];
+
+if ( ! subCommandParser) return console.log(usages);
```

Comment:
what if the user just runs `meta git remote` won't this line cause the usages to be displayed as there is no sub-command? Is that what you want or do you want to allow the user to run just `git remote`?